noir-projects/aztec-nr/aztec/src/context/avm_context.nr

```diff
@@ -201,6 +198,15 @@ impl ContextInterface for AVMContext {
         assert(false, "'get_args_hash' not implemented!");
         0
     }
+    fn nullifier_exists(self, nullifier: Field) -> bool {
+        nullifier_exists(nullifier) == 1
+    }
+    fn nullifier_valid_inclusion(self, nullifier: Field) -> bool {
+        self.nullifier_exists(nullifier) == false
```

Comment:
This seems wrong. Or maybe I'm just cooked. 

- If `nullifier_exists` returns `false` then the inclusion should be failing. 
- If `nullifier_exists` returns `true` then the inclusion should pass and the `non_inclusion` should fail.